qml/pages/Settings.qml

```diff
@@ -11,7 +11,7 @@ Item {
     ConfigurationGroup {
         id: globalSettings
         path: "/uk/co/piggz/harbour-advanced-camera"
-        property int cameraCount: 2
+        property int cameraCount: QtMultimedia.availableCameras.length
         property string cameraId: "primary"
```

Comment:
Should this be also changed to match the new cameraId format?

Reply:
True that makes sense